README.md

```diff
@@ -17,19 +17,19 @@ Table of Contents
     * [License](#license)
 
 ## Usage
-Add it as an npm dev-dependency
+Add it as an npm devDependency, as well as all the necessary [peerDependencies](https://nodejs.org/en/blog/npm/peer-dependencies/).
 
 ```bash
-$ npm i eslint-config-commercetools --save-dev
+$ npm install --save-dev eslint-config-commercetools eslint@^#.#.# eslint-config-airbnb@^#.#.# eslint-plugin-jsx-a11y@^#.#.# eslint-plugin-import@^#.#.# eslint-plugin-react@^#.#.#
```

Comment:
When I develop a library I setup already the dependencies. This "script" is just a help to install / add them to the library's package.json. This is basically one-time thing to do.
Once it's setup, whoever wants to contribute to that library doesn't care about using this "script" to install the dependencies, because they are already declared in the library's package.json.

Does it make sense now? :)

I think you are confused by this "help script". It's not required in any way to run the library. It's just a shortcut for adding the necessary dependencies to the library's package.json. That's it.